src/utils/hooks.ts

```diff
@@ -54,3 +55,36 @@ export const useAuctionTimer = (schedule: LiquidationSchedule | null) => {
     ? `in ${minutesUntilNextAuction}m ${secondsUntilNextAuction}s`
     : '';
 };
+
+/**
+ * Polls the total number of active vaults if configured to use mainnet,
+ * otherwise returns null.
+ *
+ * This fetches from subquery, rather than a vstorage query to a network node,
+ * because the query is more complex. However, subquery only works on mainnet,
+ * hence why it checks the network config.
+ */
+export const useTotalActiveVaultsQuery = (networkConfigUrl: string) => {
```

Comment:
please mention that it's expected to lag no more than 15 minutes.

should that go in the UI somewhere?

Reply:
Might be a good idea, I'll ask @otoole-brendan but won't block this PR on it

Reply:
don't think the lag disclaimer is necessary @turadg - this metric is informational only to give users some high-level stats. That it's not immediately updated is an acceptable experience for what it's designed for